crowdsec-docs/docs/bouncers/cloudflare.mdx

```diff
@@ -230,27 +257,146 @@ This only creates the required IP lists and firewall rules at cloudflare and exi
 
 Example Usage:
 ```bash
-/usr/local/bin/crowdsec-cloudflare-bouncer -s 
+sudo  crowdsec-cloudflare-bouncer -s 
 ```
 
 ### Cloudflare Cleanup
 
-This deletes all IP lists and firewall rules at cloudflare which were created by the bouncer. It also deletes the local cache. 
+This deletes all IP lists and firewall rules at cloudflare which were created by the bouncer.
 
 Example Usage:
 ```bash
-/usr/local/bin/crowdsec-cloudflare-bouncer -d 
+sudo crowdsec-cloudflare-bouncer -d 
 ```
 
-# How it works
+## How it works
 
 The service polls the CrowdSec Local API for new decisions. It then makes API calls to Cloudflare
 to update IP lists and firewall rules depending upon the decision.
 
+## Configuration Reference
+
+### `crowdsec_lapi_url`
+
+The URL of CrowdSec LAPI. It should be accessible from whichever network the bouncer has access.
```

Comment:
Wording a bit weird. Maybe just `The bouncer must be able to connect to this URL` ?